docs/dev-guide/contributing/dev_setup.rst

```diff
@@ -10,7 +10,7 @@ For the Impatient
 These instructions will create a developer install of Pulp on a dedicated
 development instance.
 
-* Start a Fedora 20 x86_64 instance that will be dedicated for development with
+* Start a RHEL 7 or Fedora 20 x86_64 instance that will be dedicated for development with
```

Comment:
@AndreaGiardini can you try with the beta repo enabled instead of the testing repo? https://repos.fedorapeople.org/repos/pulp/pulp/beta/2.6/fedora-21/x86_64/